app/config/config_test.yml

```diff
@@ -15,7 +15,6 @@ framework:
 doctrine:
     dbal:
         dbname: "%database_name%_test"
-        driver_class: Sylius\Bundle\CoreBundle\Test\MySqlDriver
```

Comment:
Sylius-Standard was upgraded before that class was renamed so it's all right :+1: